torch/fx/experimental/proxy_tensor.py

```diff
@@ -914,6 +947,31 @@ def dispatch_trace(
         concrete_args: Optional[Tuple[Any, ...]] = None,
 ) -> GraphModule:
     graph = tracer.trace(root, concrete_args)
+
+    # NB: be careful not to DCE .item() calls
+    def impure_pred(n: fx.Node) -> bool:
+        if n.target in (
+            # TODO: dedupe with accessor node in output_graph.py
+            torch.ops.aten.sym_size,
+            torch.ops.aten.sym_size.default,
+            torch.ops.aten.sym_size.int,
+            torch.ops.aten.sym_stride,
+            torch.ops.aten.sym_stride.default,
+            torch.ops.aten.sym_stride.int,
+            torch.ops.aten.sym_storage_offset,
+            torch.ops.aten.sym_storage_offset.default,
+            # TODO: sync this back to output_graph
```

Comment:
We doing the TODOs now or later?

Reply:
I'll do them before landing, but the PR has grown a lot of stuff so I think more high level directional feedback is good right now